snorkel/labeling/model/wrapper.py

```diff
@@ -0,0 +1,191 @@
+from typing import Any, List, Optional
+
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin  # type: ignore
```

Comment:
what's going on here?

Reply:
We need methods like get and set params  from these scikit learn base estimators

Reply:
I meant why is mypy upset?

Reply:
`snorkel/labeling/model/wrapper.py:4: error: No library stub file for module 'sklearn.base'`

Reply:
ah ok needs to be added to setup.cfg a la https://github.com/snorkel-team/snorkel/blob/master/setup.cfg#L92